src/Model/Attributes.php

```diff
@@ -223,4 +221,32 @@ public function __unset($key)
     {
         unset($this->attributes[$key]);
     }
+
+    /**
+     * Verify if model has a mutator method defined.
+     *
+     * @param mixed $key    attribute name
+     * @param mixed $prefix method prefix to be used
+     *
+     * @return bool
+     */
+    protected function hasMutatorMethod($key, $prefix)
```

Comment:
```suggestion
    protected function hasMutatorMethod($key, $prefix): bool
```

Reply:
typehints and return types will be reviewed in other PR.